src/components/CustomInput/CustomInput.tsx

```diff
@@ -91,37 +91,37 @@ function InputSearch(props: TInputProps, ref: TInputComponentRef) {
   return (
     <div className={classNames(containerClass)}>
       <div className={scss.inputWrapper}>
-        <label htmlFor={props.name} className={classNames(labelClass)}>
+        <label className={classNames(labelClass)} htmlFor={props.name}>
           {props.placeholder}
         </label>
         <div className={scss.iconLeft}>
-          <img src={leftIcon} />
+          <Icon icon="mag" />
         </div>
         <input
-          required
-          readOnly={props.readOnly}
-          type={props.type}
+          className={classNames(inputClass)}
           id={props.name}
           name={props.name}
-          className={classNames(inputClass)}
-          placeholder={props.placeholder}
           onBlur={(e) => onInputBlur(e)}
```

Comment:
nesse caso nao precisa passar explicitamente o evento pra função, o react ja faz isso implicitamente pra gente
```suggestion
          onBlur={onInputBlur}
```